flask_example/config.example.py

```diff
@@ -54,3 +54,10 @@
 
 BATTLENET_CLIENT_ID = ''
 BATTLENET_CLIENT_SECRET = ''
+
+BOX_CLIENT_ID = ''
+BOX_CLIENT_SECRET = ''
+BOX_ACCESS_TOKEN_PARAMS = {
```

Comment:
Why do you need this `BOX_ACCESS_TOKEN_PARAMS`?

Reply:
The Box access token endpoint requires the client ID and client secret when exchanging the authorization code for an access token. 

https://developer.box.com/reference#token

Reply:
that is because Box only support `client_secret_post` method, try it with something like https://github.com/authlib/loginpass/blob/master/loginpass/slack.py#L41